src/js/Manager/ClipboardManager.js

```diff
@@ -0,0 +1,90 @@
+import ErrorManager from '@js/Manager/ErrorManager';
+import SettingsService from '@js/Services/SettingsService';
+
+class ClipboardManager {
+
+    /**
+     *
+     * @return {String}
+     */
+    async readText() {
+        try {
+            var element = this._CreateDOMElement();
+            await document.execCommand('paste');
+            var result = element.value
+            this._RemoveDOMElement(element);
+            return result;
+        } catch (e) {
+            ErrorManager.logError(e, "ClipboardManager.readText()");
+        }
+    }
+
+    /**
+     *
+     * @param {String} type
+     * @param {String} value
+     */
+    write(type, value) {
+        if(type === "password") {
+            this.writePassword(value);
+        }
+        else {
+            this.writeText(value);
+        }
+    } 
+
+    /**
+     *
+     * @param {String} value
+     * @param {String} type
+     */
+    writeText(value) {
+        try {
+            var element = this._CreateDOMElement(value);
+            document.execCommand('copy', false, element.value);
+            this._RemoveDOMElement(element);            
```

Comment:
Why can't we use `navigator.clipboard.writeText(value);` here?

On the main app i had to add [this](https://github.com/marius-wieschollek/passwords/blob/master/src/js/Classes/Utility.js#L8) so i could copy from a text field because of browser restrictions. Could the same happen here?

Reply:
The web API is working, I changed it. 
Regarding your suggested implementation. I did nearly the same by creating a virtual element on the background page and setting focus on it. But there is no need for the range and selection part so far I know. I tested it already on Chrome, Firefox and Edge-chromium. I will just copy the `readOnly` flag for the virtual element for security reasons 😉

Reply:
It seams that the web api is not working on edge-chromium. I get DOM object not focused messages. I think this was also the reason why I choosed document.execCommand after setting focus to the element. This works in all browsers.

Reply:
@marius-wieschollek the api is not proper implemented in the background script see https://bugs.chromium.org/p/chromium/issues/detail?id=874848. For this reason we can not use another function than document.execCommand within the background script. If you have another idea please let me know.